improver/standardise.py

```diff
@@ -52,6 +52,41 @@
 class StandardiseMetadata(BasePlugin):
     """Plugin to standardise cube metadata"""
 
+    @staticmethod
+    def _rm_air_temperature_status_flag(cube: Cube) -> Cube:
+        """
+        Remove air_temperature status_flag coord by applying as NaN to cube data.
+
+        See https://github.com/metoppv/improver/pull/1839 for further details.
+        """
+        coord_name = "air_temperature status_flag"
+        try:
+            coord = cube.coord(coord_name)
+        except CoordinateNotFoundError:
+            coord = None
```

Comment:
If you do this, you don't need the next `if` statement.
```suggestion
            return cube
```